### PR TITLE
Robons bug fix

### PR DESCRIPTION
**Pull request by robons**
- Ensures that you can create attribute values from a column containing missing values (attributes are not required by default).
- Ensure that the observation's columns in the CSV-W point to the correct propertyUrl for the measure dimension/constant measure
- Ensure that the csvqb .whl package actually contains some code.

### CODE REVIEW

**Review by santhosh-thangavel — APPROVED** [review]

File: csvqb/csvqb/inputs.py [hunk 1]
```diff
@@ -28,20 +28,39 @@ def pandas_input_to_columnar(
...
 
-def pandas_input_to_columnar_str(
-    maybe_columnar_data: PandasDataTypes, allow_none: bool = True
-) -> Iterable[str]:
-    series = pandas_input_to_columnar(maybe_columnar_data, allow_none)
+def pandas_input_to_columnar_optional_str(
+    maybe_columnar_data: PandasDataTypes,
+    allow_no_data_at_all: bool = True,
+) -> Iterable[Optional[str]]:
+    series = pandas_input_to_columnar(maybe_columnar_data, allow_no_data_at_all)
```
Comment (canwaf) [inline csvqb/csvqb/inputs.py:44]: I know where this is being used but what is the actual usecase? We don't allow sparse columns.

File: devtools/devtools/behave/csv2rdf.py [hunk 2]
```diff
@@ -15,11 +15,13 @@
 from devtools.helpers.tar import dir_to_tar, extract_tar
 from devtools.behave.temporarydirectory import get_context_temp_dir_path
 
+client = docker.from_env()
+client.images.pull("gsscogs/csv2rdf")
```
Comment (canwaf) [inline devtools/devtools/behave/csv2rdf.py:19]: 🙌

**Review by canwaf — APPROVED** [review]
All sensible changes. Approved.

File: csvqb/csvqb/inputs.py (hunk 1 above)
Comment (robons) [reply csvqb/csvqb/inputs.py:44]: Just confirming, we do allow sparse attributes.

**Pull request merged**